packages/headless/src/controllers/sort/headless-sort.ts

```diff
@@ -69,13 +69,12 @@ function getCriterionAsArray(state: Partial<SortInitialState>) {
   return isArray(state.criterion) ? state.criterion : [state.criterion];
 }
 
-/** The `Sort` controller allows changing how the results are sorted.*/
+/** The `Sort` controller manages how the results are sorted.*/
```

Comment:
`/** The `Sort` controller manages how the results are sorted. */`
(Add space at end between . and *)